spec/spec_helper.rb

```diff
@@ -67,6 +67,22 @@ def expect_shell_command(*command, exitstatus: 0, output: '')
   expect(Open3).to receive(:popen2e).with(*command).and_yield(mock_input, mock_output, mock_thread)
 end
 
+# If the `described_class` of a spec is a `Fastlane::Action` subclass, it runs
+# it with the given parameters.
+#
+def run_described_action(parameters, lane_name = 'test')
```

Comment:
Also for clarity I'd probably name this helper method `run_described_fastlane_action` (especially given the term "action" can be interpreted and read differently depending on the context the reader is in when seeing this), wdyt?

Reply:
See [`635f9ca` (#330)](https://github.com/wordpress-mobile/release-toolkit/pull/330/commits/635f9cad73653fb4db59a7962a25981616473283)